_data/simple-icons.json

```
@@ -8011,6 +8011,12 @@
             "hex": "FF6600",
             "source": "https://ktm.com"
         },
+        {
+            "title": "ktor",
```

I think we should use a capital `K` here:
```suggestion
            "title": "Ktor",
```